docs/review_process.md

```diff
@@ -61,9 +61,9 @@ Alternatively, just [open a new issue](https://github.com/conan-io/conan-center-
 
 ## Avoiding conflicts
 
-It's recommended to rebase your changes on top of the master branch to avoid conflicts.
 Right now, neither GitHub itself nor conan-center-bot notify about merge conflicts, so it's the contributor's responsibility to periodically check for the conflicts.
 Obviously, PRs that have merge conflicts are never merged, and all the conflicts have to be resolved first.
+It's recommended to [synchronize your branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) to solve avoid conflicts.
```

Comment:
```suggestion
Please [synchronize your branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) to take into account the latest changes in the main branch.
```

We cannot enforce this enough. We would really like to avoid any _merge_ (for the same `conanfile.py`) in `master` as it generates a new recipe revision... and we need to build all packages in that merge (it can fail, we don't merge more packages until it finishes,...)